core/src/replay_stage.rs

```diff
@@ -4139,7 +4139,6 @@ impl ReplayStage {
                 )
             },
         )
-        .expect("rooting must succeed")
```

Comment:
you can revert this, agave master was changed to remove `SetRootError`, we should continue to `expect` here

Reply:
ah got it, carry on